aphrodite/modeling/sampling_metadata.py

```diff
@@ -683,7 +720,7 @@ def from_lists(cls, temperatures: List[float], top_ps: List[float],
 
     @staticmethod
     def _get_sequence_seeds(
-        seed: int,
```

Comment:
What's the reason for this?

Reply:
It's just correcting the typehint.

`seed` comes from an `int|None` source, and this func has explicit behavior for the `None` case.